changePassword.js

```diff
@@ -0,0 +1,24 @@
+function next() {
+    let firstField = document.getElementsByClassName('pass');
+    let secondField = document.getElementsByClassName('conf-pass');
```

Comment:
It is useful to use `let` when it is not possible to use `const`, so here variables should be declared with `const`

Reply:
But in this case I want to change one of the properties of these elements. That's why I used **let**.

Reply:
HTML element is structural type. So you can declare button as a constant and change the inner properties. Const protects you from unintentional change of the reference of the object.